igvm_defs/src/lib.rs

```diff
@@ -597,13 +602,15 @@ pub struct IGVM_VHS_RELOCATABLE_REGION {
 /// is.
 ///
 /// # Architecture
-/// X64 only.
+/// X64 and ARM64.
 #[repr(C)]
 #[derive(Copy, Clone, Debug, AsBytes, FromBytes, FromZeroes, PartialEq, Eq)]
 pub struct IGVM_VHS_PAGE_TABLE_RELOCATION {
     /// Compatibility mask.
     pub compatibility_mask: u32,
-    /// VP Index for paging information, like CR3 and EFER.
+    /// VP Index for paging information.
+    /// On X64 these registers include CR3, CR4, RIP and GDTR
```

Comment:
We only use `vp_index` to search for paging information (paging enabled via efer and cr3 for example), not RIP/GDTR/PC. Those are tied to the other relocation region.